Documentation/operations/upgrade.rst

```diff
@@ -434,9 +434,8 @@ functional in general. The following is a list of operations that will not be
 available during the upgrade:
 
 * API aware policy rules are enforced in user space proxies and are currently
```

Comment:
Grammar nits, omit "currently", yes changes are in scope for the PR even though you didn't technically touch this line
```suggestion
* API-aware policy rules are enforced in user space proxies and are
```